src/program.js

```diff
@@ -408,7 +410,8 @@ Example: $0 --help run.
         type: 'boolean',
         default: false,
       },
-    });
+    })
+    .command('docs', 'Open the web-ext documentation', commands.docs, {});
```

Comment:
maybe let's make this more explicit: "Open the web-ext documentation in a browser"